makefile

```diff
@@ -163,7 +163,7 @@ prefs: # Build the preferences web app
 
 sdist: manpage prefs # Build a source tarball
 	@set -euo pipefail
-	# See https://github.com/Ulauncher/Ulauncher/pull/1337 for why we're not using setuptools
+	# See https://github.com/Ulauncher/Ulauncher/pull/1337 for why we're not using setuptools directly
```

Comment:
This change makes no sense. Did you actually read the job name, the code and the link I put in before you changed this? :thinking:

Reply:
AI added that. It made sense to me, so I kept it.
I will revert it. No big deal.